broken grpc build

@matt-kwong could you please take a look again?

See https://oss-fuzz-build-logs.storage.googleapis.com/log-711a87ff-29fd-42e2-81ec-98589a520e8f.txt

```
Step #2: bazel-bin/libgrpc_resolver_dns_ares.a(grpc_ares_wrapper.pic.o): In function `grpc_resolve_address_ares_impl':
Step #2: /root/.cache/bazel/_bazel_root/ec9ee79d770c6b7183036e592ac23980/execroot/grpc/src/core/ext/resolver/dns/c_ares/grpc_ares_wrapper.c:248: undefined reference to `ares_gethostbyname'
Step #2: /root/.cache/bazel/_bazel_root/ec9ee79d770c6b7183036e592ac23980/execroot/grpc/src/core/ext/resolver/dns/c_ares/grpc_ares_wrapper.c:250: undefined reference to `ares_gethostbyname'
Step #2: bazel-bin/libgrpc_resolver_dns_ares.a(grpc_ares_wrapper.pic.o): In function `on_done_cb':
Step #2: /root/.cache/bazel/_bazel_root/ec9ee79d770c6b7183036e592ac23980/execroot/grpc/src/core/ext/resolver/dns/c_ares/grpc_ares_wrapper.c:182: undefined reference to `ares_strerror'
Step #2: /root/.cache/bazel/_bazel_root/ec9ee79d770c6b7183036e592ac23980/execroot/grpc/src/core/ext/resolver/dns/c_ares/grpc_ares_wrapper.c:157: undefined reference to `ares_inet_ntop'
Step #2: /root/.cache/bazel/_bazel_root/ec9ee79d770c6b7183036e592ac23980/execroot/grpc/src/core/ext/resolver/dns/c_ares/grpc_ares_wrapper.c:172: undefined reference to `ares_inet_ntop'
Step #2: bazel-bin/libgrpc_resolver_dns_ares.a(grpc_ares_wrapper.pic.o): In function `grpc_ares_init':
...
```

--- a/projects/grpc/build.sh
+++ b/projects/grpc/build.sh
@@ -42,6 +42,7 @@ bazel-bin/test/core/util/*.a \
 bazel-bin/test/core/end2end/*.a \
 bazel-bin/third_party/boringssl-with-bazel/libssl.a \
 bazel-bin/third_party/boringssl-with-bazel/libcrypto.a \
+bazel-bin/external/submodule_cares/_objs/ares/external/submodule_cares/cares/*.o \
 bazel-bin/external/submodule_zlib/_objs/z/external/submodule_zlib/*.o \
 bazel-bin/third_party/nanopb/*.a \
 bazel-bin/*.a \
@@ -54,7 +55,7 @@ EXTERA_BAZEL_FLAGS="--strip=never  $(for f in $CXXFLAGS; do if [ $f != "-stdlib=
 bazel build --dynamic_mode=off --spawn_strategy=standalone --genrule_strategy=standalone \
   $EXTERA_BAZEL_FLAGS \
   $NO_VPTR \
-	:all test/... third_party/boringssl-with-bazel/... third_party/nanopb/...
+  :all test/... third_party/boringssl-with-bazel/... third_party/nanopb/...
 
 CFLAGS="${CFLAGS} -Iinclude -I."
 CXXFLAGS="${CXXFLAGS} -Iinclude -I. -stdlib=libc++"

